SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -219,8 +219,8 @@
 
     <!-- Calculate a project-specific directory name -->
     <PropertyGroup>
-      <FolderDisambiguator Condition=" $(FolderDisambiguator)=='' ">_$(Platform)_$(Configuration)_$([System.DateTime]::Now.Ticks)</FolderDisambiguator>
-      <ProjectSpecificDir>$(SonarQubeOutputPath)\$(MSBuildProjectName)_$(FolderDisambiguator)</ProjectSpecificDir>
+      <FolderDisambiguator Condition=" $(FolderDisambiguator)=='' ">_$(Platform)_$(Configuration)_$([System.DateTime]::Now.ToString("ffff"))</FolderDisambiguator>
+      <ProjectSpecificDir>$(SonarQubeOutputPath)\$(MSBuildProjectName)$(FolderDisambiguator)</ProjectSpecificDir>
```

Comment:
The new version uses the ten-thousands of a second part of the DateTime instead of the full number of Ticks.
Also, there was a redundant "_" being added on line 223.

The old and new names look as follows:
Old: XXX__AnyCPU_Debug_635804433148301490
New: XXX_AnyCPUDebug_8301

The new version is still as readable but sixteen characters shorter. The chance of a name collision is higher than it was before but still sufficiently unlikely.